docs/client-metrics.md

```diff
@@ -200,6 +200,8 @@ The event stream is a log of events and the time they occurred while the user is
 * signup.password.hidden - password is hidden
 * signup.email-optin.visible.true - email opt-in is visible
 * signup.email-optin.visible.false - email opt-in is not visible
+* signup.checkbox.change.marketing-email-optin.checked - user checked the email opt-in
+* signup.checkbox.change.marketing-email-optin.unchecked - user unchecked the email opt-in
```

Comment:
I guess we should document what the content server sends, but that could make it harder to find in DataDog